integration/test/basic/basic_test.go

```diff
@@ -57,3 +58,46 @@ func TestChartOperatorBootstrap(t *testing.T) {
 		t.Fatalf("bad release status, want %q, got %q", expectedStatus, actualStatus)
 	}
 }
+
+// TestChartConfigChartsInstalled checks that the charts for any chartconfig
+// CRs installed in the cluster have been deployed.
+func TestChartConfigChartsInstalled(t *testing.T) {
+	guestNamespace := "giantswarm"
+	logger, err := micrologger.New(micrologger.Config{})
+	if err != nil {
+		t.Fatalf("could not create logger %v", err)
+	}
+
+	guestG8sClient := g.G8sClient()
+	chartConfigList, err := guestG8sClient.CoreV1alpha1().ChartConfigs(guestNamespace).List(metav1.ListOptions{})
+	if err != nil {
+		t.Fatalf("could not get chartconfigs %v", err)
+	}
+
+	if len(chartConfigList.Items) > 0 {
```

Comment:
When this is true? Is there a race? I'm afraid when `len(chartConfigList.Items) == 0` this tests nothing.

Reply:
The problem is the number of chartconfigs depends on the version bundle version. The current vb has 0 and the WIP vb has 1. But this could lead to a race so I'll update the test to use the version bundle version.